tests/src/readytorun/tests/mainv1.csproj

```diff
@@ -11,6 +11,8 @@
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
     <CLRTestKind>BuildAndRun</CLRTestKind>
     <DefineConstants>$(DefineConstants);STATIC;CORECLR</DefineConstants>
+    <!-- https://github.com/dotnet/coreclr/issues/23643 -->
+    <CrossGenTest Condition=" '$(BuildOS)' == 'Windows_NT' And ('$(BuildArch)' == 'x64' Or '$(BuildArch)' == 'x86') ">false</CrossGenTest>
```

Comment:
Refering to my comment, i think this should be set to false unconditionally to exclude the test from the R2R mode, given how special this test is